### PR TITLE
Schedule Feature V1.2.1 Branch

### PR DESCRIPTION
**Pull request by zhihong8888**
Added:
-Unique Schedule list exception
-Schedule list Ui
-check Employee Id exists, before allowing user to add a new schedule with the specified employee id
-date model updated

### CODE REVIEW

File: src/main/java/seedu/address/model/person/exceptions/PersonNotFoundException.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package seedu.address.model.person.exceptions;
 
 /**
- * Signals that the operation is unable to find the specified person.
+ * Signals that the operation is unable to find the specified Schedule.
```
Comment (LimYiSheng) [inline src/main/java/seedu/address/model/person/exceptions/PersonNotFoundException.java]: Did you use reflector for this? =O

Comment (zhihong8888) [reply src/main/java/seedu/address/model/person/exceptions/PersonNotFoundException.java]: yes..i'll change it back. that's an unintentional change.

Comment (zhihong8888) [reply src/main/java/seedu/address/model/person/exceptions/PersonNotFoundException.java]: nvm i realised i edited it.. i wasn't due to reflector but i edited that wrong file just now

File: src/main/java/seedu/address/model/person/Person.java [hunk 2]
```diff
@@ -112,6 +129,19 @@ public boolean isSamePerson(Person otherPerson) {
                 && (otherPerson.getPhone().equals(getPhone()) || otherPerson.getEmail().equals(getEmail()));
     }
 
+    /**
+     * Returns true if both persons of the same name have at least one other identity field that is the same.
+     * This defines a weaker notion of equality between two persons.
+     */
```
Comment (LimYiSheng) [inline src/main/java/seedu/address/model/person/Person.java]: Do edit the comment above, i guess it's supposed to be both persons of the same employeeId

**Comment by XiiaoPanda** [comment]
Command and UI work beautifully.

**Pull request merged**